drivers/Makefile.dep

```diff
@@ -202,6 +202,12 @@ ifneq (,$(filter hih6130,$(USEMODULE)))
   FEATURES_REQUIRED += periph_i2c
 endif
 
+ifneq (,$(filter hmc5883l,$(USEMODULE)))
+  FEATURES_REQUIRED += periph_i2c
+  FEATURES_REQUIRED += periph_gpio_irq
```

Comment:
this feature is only required if the actual interrupt submodule is used, right?

Reply:
Yes, I just missed this dependence here when I introduced `hmc5883l_int`. Thanks.

Reply:
Fixed